analyze_issue_metadata.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import json
+import os
+import sys
+
+from datetime import datetime as dt
+
+import github3
+
+import get_issue_metadata as gim
+
+
+def main(state):
+
+    """
+    Command line tool that reads in an input file
+    containing GitHub repository names, calls read_issue_metadata
+    and returns a JSON file of issue data.
+
+    Usage (command line):
+        $ python analyze_issue_metadata.py googlecloudplatform repos.txt
+
+    Args:
+        state: GitHub state of issue ('open', 'closed'
+               or 'all')
+        sys.argv[1]: github owner or organization
+        sys.argv[2]: path to input file containing names of repos belonging to
+                     owner or org in sys.argv[1].  One repo name per line.
+
+    Returns:
+        A JSON of issue metadata
+
+    Raises:
+        N/A
+    """
+
+    org = str(sys.argv[1])
+
+    with open(sys.argv[2]) as f:
+        repos = f.readlines()
+    repos = [x.strip() for x in repos]
+
+    # Authenticate with GitHub using Personal Access Token
+    g = github3.login(token=os.environ['GH_TOKEN'])
```

Comment:
Not necessary right now, but a valuable future enhancement might be to accept a `--github-token` CLI parameter. If you are interested in doing this, look at [argparse](https://docs.python.org/3/library/argparse.html#module-argparse), which makes it easy.